### DISCUSSION TITLE
Is there support for fluentbit?

### DISCUSSION THREAD

**Discussion by SubhraB** [gravitational/teleport#22093] [Q&A] [answered]
I started looking at this documentation for forwarding access logs with fluentd. Wanted to check  if we can do it with fluentbit. Since we are using fluentbit for forwarding all our logs from EKS application via fluentbit, it will be easier if we can do the same for teleport.

We need to ingest these logs in other backends, eg. OpenSearch, Kafka etc. I think we can use the teleport event handler with any http endpoint. But not sure if that will work with Kafka as well.

**Comment by SubhraB** [comment]
I am now trying to use the fluentbit http input plugin, I think it should work.

**Comment by SubhraB** [comment, marked as answer]
It all worked at the end, using the http input plugin. However, I couldn't make the event handler cleanly run on non-tls mode. I was running the event handler with fluentbit as sidecar in kubernetes. I was able to make it work on non-tls mode. But I was still forced to configure  and mount the certs and keys in the toml file, even though they are of no use. 

Closing this discussion for now.

**Comment by karendarm11** [comment]
@SubhraB how did you make it work in non-tls mode?
